lib/generate.m

```diff
@@ -0,0 +1,61 @@
+function [x] = generate(fs,t,q,friv,w,taus,a0,a)
+%generate is the final tool that helps us in generating sigma and delta
+%(longitudinal and transversal) signals related to the Schottky analysis
```

Comment:
maybe you could add some comment lines explaining meaning/type of input and output parameters